src/Tribe/Venue.php

```diff
@@ -451,14 +451,6 @@ public function create( $data, $post_status = 'publish', $avoid_duplicates = fal
 
 				$this->save_meta( $venue_id, $data );
 
-				/**
```

Comment:
Intentional?

Reply:
Yes, intentional. The action would be fired twice with the same parameters.
My guess is that was the result of a merge.